lib/vcloud/launcher/launch.rb

```diff
@@ -5,11 +5,11 @@ class Launch
       class MissingPreambleError < RuntimeError ; end
       class MissingConfigurationError < RuntimeError ; end
 
-      attr_reader :config, :cli_options
+      attr_reader :config, :options
 
-      def initialize(config_file, cli_options = {})
```

Comment:
I quite like the distinction that `config_file` is required whereas everything else is optional.